assembl/static2/js/app/components/debate/thread/answerForm.jsx

```diff
@@ -55,9 +59,22 @@ class AnswerForm extends React.PureComponent<*, AnswerFormProps, AnswerFormState
   };
 
   handleInputFocus = () => {
-    const isUserConnected = getConnectedUserId(); // TO DO put isUserConnected in the store
-    if (!isUserConnected) {
-      inviteUserToLogin();
+    const { debateData, identifier, handleAnswerClick } = this.props;
+    const isPhaseCompleted = getIfPhaseCompletedByIdentifier(debateData.timeline, identifier);
+    if (isPhaseCompleted) {
+      const body = (
+        <div>
+          <Translate value="debate.noAnswer" />
+        </div>
+      );
+      displayModal(null, body, true, null, null, true);
+    } else {
+      const isUserConnected = getConnectedUserId(); // TO DO put isUserConnected in the store
```

Comment:
you should use the `promptForLoginOr` util here so we wont have to search this code everywhere if the TODO is adressed one day